pkg/system/syscall_windows.go

```diff
@@ -66,6 +67,22 @@ func IsWindowsClient() bool {
 	return osviex.ProductType == verNTWorkstation
 }
 
+// IsIoTCore returns true if the currently running image is based off of
+// Windows 10 IoT Core.
+// @engine maintainers - this function should not be removed or modified as it
+// is used to enforce licensing restrictions on Windows.
```

Comment:
`enforce` may be a strong term here, since obviously anyone could remove this code and build a private version of Docker (or just call HCS APIs directly).